meta.yaml

```diff
@@ -23,9 +23,9 @@ requirements:
     - setuptools_scm
   run:
     - python
-    - greenlet >=0.4
-    - pyee >=8.0.1
-    - websockets >=8.1
+    - greenlet==1.1.2
+    - pyee==8.1.0
```

Comment:
8.1.0 is the last version available on anaconda (after that it's broken). I'll create an issue once its merged to align them once pyee is up2date again on anaconda.